pkg/etcdmgr/bootstrap/client.go

```diff
@@ -58,7 +54,7 @@ func GetEtcdClients(configDir, token, ipAddr, nodeID string) ([]string, error) {
 	factory := EmbeddedEtcdFactory{}
 	ee, err := factory.NewEmbeddedEtcd(token, conf, true)
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("failed to start embedded etcd. %+v", err)
```

Comment:
so the NewEmbeddedEtcd func actually "starts" the etcd server as well? (i.e. is this error message accurate?)